package.json

```diff
@@ -4,29 +4,29 @@
   "description": "%CUSTOM_PLUGIN_SERVICE_DESCRIPTION%",
   "main": "dist/index.js",
   "scripts": {
-    "checkonly": "! grep -R '\\.only' tests/",
-    "coverage": "npm run unit -- --coverage-report=text-summary",
-    "postcoverage": "tap --coverage-report=lcov --no-browser",
+    "checkonly": "! grep -R '\\\\.only' __tests__/",
+    "coverage": "npm run unit -- --coverage",
```

Comment:
Please make sure that this script generates a `lcov` output file, if not add back the `postcoverage` with proper edits to work with jest

Reply:
`lcov` report is already generated